app/User/ReverseProxyUserProvider.php

```diff
@@ -134,7 +140,12 @@ public function getName()
      */
     public function getEmail()
     {
-        return REVERSE_PROXY_DEFAULT_DOMAIN !== '' ? $this->username.'@'.REVERSE_PROXY_DEFAULT_DOMAIN : '';
+	if (! REVERSE_PROXY_MAIL_HEADER ){
+	        return REVERSE_PROXY_DEFAULT_DOMAIN !== '' ? $this->username.'@'.REVERSE_PROXY_DEFAULT_DOMAIN : '';
+	}else{
```

Comment:
That would be appreciated if you could indent your code the same way as the rest to be more consistent.